codeception.yml

```diff
@@ -5,7 +5,7 @@ paths:
     support: tests/support
     envs: tests/envs
 settings:
-    shuffle: true
```

Comment:
You are trying to change the value of shuffle again. I got used to shuffle: true already.